src/screens/NewAnalytics/PaymentAnalytics/PaymentsProcessed/PaymentsProcessed.res

```diff
@@ -69,10 +97,26 @@ let make = (
     None
   }, [])
 
+  let totalAmount =
+    paymentsProcessed
+    ->getArrayFromJson([])
+    ->getValueFromArray(0, JSON.Encode.array([]))
```

Comment:
why default value is `JSON.array` shoudn't it be `JSON.object`

Reply:
updated file